code/modules/atmospherics/machinery/components/gas_recipe_machines/crystallizer.dm

```diff
@@ -109,8 +109,7 @@
 
 ///Checks if the reaction temperature is inside the range of temperature + a little deviation
 /obj/machinery/atmospherics/components/binary/crystallizer/proc/check_temp_requirements()
-	var/datum/gas_mixture/contents = airs[2]
-	if(contents.temperature >= selected_recipe.min_temp * MIN_DEVIATION_RATE && contents.temperature <= selected_recipe.max_temp * MAX_DEVIATION_RATE)
```

Comment:
What's going on here?

Reply:
instead of using the internal temperature to check if the recipe requirement were met (so the internal gases must be on that temperature to work) it checked the port temperature. that's fixed (and also that's why i increased the amount of cooling gas allowed to cool)